package.json

```diff
@@ -30,6 +30,7 @@
     "@types/styled-components": "5.1.2",
     "@typescript-eslint/eslint-plugin": "3.8.0",
     "@typescript-eslint/parser": "3.8.0",
+    "@zendeskgarden/container-utilities": "^0.5.4",
```

Comment:
Looks like you added this via `npm` rather than `yarn`.

- remove `package-lock.json` (it should not appear in this PR's change set)
- run `yarn add @zendeskgarden/container-utilities@0.5.4 --dev`

In the end you should have a pinned `0.5.4` version (no "hat") and an updated `yarn.lock`.

Reply:
Doh, ty!